server/src/main/kotlin/core/gameflow/showdown.kt

```diff
@@ -4,3 +4,42 @@ sealed class ShowdownAction
 
 data class ShowCards(val playerId: Int) : ShowdownAction()
 data class MuckCards(val playerId: Int) : ShowdownAction()
+
+fun resolveShowdown(handState: HandState): List<ShowdownAction> {
+
+    val orderedPlayers = when {
+        handState.lastAggressor == null -> handState.orderedPlayers(handState.buttonPosition + 1)
+        else -> handState.orderedPlayers(handState.lastAggressor.position)
+    }.filter { it.isInGame }
+
+    // when only one player is left in the game
+    if (orderedPlayers.size == 1)
+        return listOf(MuckCards(orderedPlayers.first().id))
+
+    // all players must show their cards if there are more streets to be dealt
+    if (handState.bettingRound != BettingRound.RIVER)
+        return orderedPlayers.map { ShowCards(it.id) }
+
+    // one player definitely wins with another if he has a better hand
+    // and he has put at least the same amount of chips into the pot
```

Comment:
Why do we need to check amount of chips here?

Reply:
Because in cases with multiple pots a player may win a side pot even if another player has the best hand overall.

Reply:
ok

Reply:
By comparing chips commitment we check if one player is playing for a side pot that the other one isn't.